### ISSUE TITLE
Storage volume still available when uploading with --use-root-swap option

### ISSUE THREAD

**Issue by ChrisBr** [SUSE/Enceladus#41]
When I upload an image with the --use-root-swap option enabled, the created ami still has the storage volume attached

```
tux@ip-42:/> lsblk
NAME   MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
sda      8:0    0  10G  0 disk 
`-sda1   8:1    0  10G  0 part /
sdf      8:80   0  20G  0 disk 
`-sdf1   8:81   0  20G  0 part /mnt
```

When I mount sdf1 it contains the image tar file and a folder with the extracted image! I guess we want to delete the storage volume after we've dumped it on the new root volume, right?

**Comment by ChrisBr** [comment]
I see from the man

```
The code will then skip the clean up and a storage volume, a target root volume and a stopped instance will remain in the users account.
```

But nevertheless we want to detach the storage volume, right?

**Comment by rjschwei** [comment]
Version 0.6.0 is building in OBS and should fix the issue. Please report back

**Comment by ChrisBr** [comment]
Hm, script doesn't work anymore :/

```
Wait for volume to detach
. 
Wait for volume to detach
.  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  . 
Wait for volume to detach
.  .  .  .  .  .  .  .  .  .  .  .  . 
Unable to detach volume
```

**Comment by rjschwei** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

On 07/01/2015 08:50 AM, Christian Bruckmayer wrote:

> Hm, script doesn't work anymore :/
> 
> `Wait for volume to detach . Wait for volume to detach .  .  .
> .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .
> .  .  .  . Wait for volume to detach .  .  .  .  .  .  .  .  .  .
> .  .  . Unable to detach volume`

Yes, of course because the volume is still mounted, duh, me needs to
slow down a bit and not do twenty things at the same time.

---

Robert Schweikert                   MAY THE SOURCE BE WITH YOU
Public Cloud Architect                         LINUX
rjschwei@suse.com
IRC: robjo
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQEcBAEBAgAGBQJVk/HwAAoJEE4FgL32d2UkiyUIAMAy+NlUIAMY0A4lpaGzDqxR
/bmb8CIMNALHdtGtyom19SD9iE7btLLDyC0naogt2o/7O5fYdakeYa1U2GCZm06w
f/qNmvLGaPM1FAo//W1OerSe0ZgCjoU9FT9WDnot8a4xH8np9l9wnXnh54/kP7x/
Jf+IO0Q9D90AL/n2tcO4BJj+XxqJpx4pOEgdku6jgGoZ3XSKCNQQxsXkbr2to8Cz
D1wa7e1TAk3PD4O0Wc3Tad8aSUJOVwmWXDZwj8z1D44QtGLAsX8xbdIBaWm9vFPJ
D5XvaDeHlTwVj3GTcLGSCkePGYYz6oix725tfMSS3c6pOtrd8B1doDzbq2XSltM=
=PMpL
-----END PGP SIGNATURE-----

**Comment by ChrisBr** [comment]
@rjschwei Yeah, I assumed sth like this!
Don't hurry, that's nothing critical ...

**Comment by rjschwei** [comment]
Now really fixed, still version 0.6.0 in OBS

**Comment by ChrisBr** [comment]
@rjschwei works perfectly fine :) Thanks!
